### ISSUE TITLE
TrackPopupMenu has the wrong return type in certain context

### ISSUE THREAD

**Issue by Craftplacer** [microsoft/CsWin32#338]
## Actual behavior

CsWin32 generates the function signature for `TrackPopupMenu` and `TrackPopupMenuEx` as `bool`, which is correct for using no parameters. The ["Return value"](https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-trackpopupmenu#return-value) section however states that the return value will be an `UINT_PTR` (`IntPtr`, `int`?) when `TPM_RETURNCMD` has been passed.

## Expected behavior

Our function sig that we have written is this:

```cs
[DllImport(nameof(User32))]
internal static extern int TrackPopupMenuEx(IntPtr hmenu, uint fuFlags, int x, int y, IntPtr hwnd, IntPtr lptpm);
```

## Repro steps

1. `NativeMethods.txt` content:

```
TrackPopupMenu
TrackPopupMenuEx
```

3. Any of your own code that should be shared?

```cs
unsafe
{
    // this turns into a type compiler error
    int clickedItemId = PInvoke.TrackPopupMenuEx(_hMenu, (uint)TRACK_POPUP_MENU_FLAGS.TPM_RETURNCMD, x, y, hWnd);
}
```

### Context

- CsWin32 version: `0.1.506-beta`
- Target Framework: `net5.0-windows`

**Comment by AEAEAEAE4343** [comment]
UINT_PTR translates to IntPtr.

**Comment by jnm2** [comment]
(Or `UIntPtr`)

**Comment by AArnott** [comment]
Fixed by #363
